packages/kbn-es/src/cluster.js

```diff
@@ -579,6 +579,8 @@ exports.Cluster = class Cluster {
    * @param {ServerlessOptions} options
    */
   async runServerless(options = {}) {
+    // Ensure serverless ES nodes are not running
+    teardownServerlessClusterSync(this._log, options);
```

Comment:
@jloleysens you can pass `kill: true` into the `runServerless` config and it will detect running nodes and kill them if needed. See https://github.com/elastic/kibana/blob/main/packages/kbn-es/src/utils/docker.ts#L330

Reply:
Nice! I had automerge on 🤦🏻  I'll fix this in a follow up PR